modules/storages/app/common/storages/peripherals/oauth_configurations/nextcloud_configuration.rb

```diff
@@ -47,8 +47,8 @@ def initialize(storage)
 
         # rubocop:enable Lint/MissingSuper
 
-        def extract_origin_user_id(rack_access_token)
-          rack_access_token.raw_attributes[:user_id]
+        def storage_name
+          "nextcloud"
         end
```

Comment:
As per @NobodysNightmare comment, you can remote this. We made `Storage#to_s` to map to the `short_provider_name`. =)